routing_common/transit_types.cpp

```diff
@@ -161,6 +161,13 @@ bool Gate::IsValid() const
 }
 
 // ShapeId ----------------------------------------------------------------------------------------
+bool ShapeId::operator<(ShapeId const & rhs) const
+{
+  if (m_stop1Id != rhs.m_stop1Id)
+    return rhs.m_stop1Id < m_stop1Id;
```

Comment:
Is it important that the order is reversed for stop1?

Reply:
It's a mistake. Fixed. Thanks.